toPDFviaQuarto.qmd

```diff
@@ -0,0 +1,169 @@
+---
+title: Poseidon package specification \vspace{-1cm}
+format:
+  pdf:
+#### general settings ####
+    #keep-tex: true
+    colorlinks: true
+#### toc settings ####
+    toc: true
+    toc-depth: 2
+    number-sections: true
+    shift-heading-level-by: -1
+#### page layout ####
+    documentclass: article
+    papersize: a4paper
+    geometry:
+      - top=30mm
+      - left=20mm
+      - heightrounded
+#### code layout ####
+    highlight-style: tango
+#### detailed configuration ####
+    include-in-header: 
+       text: |
+         % wrap long code lines
+         \usepackage{fvextra}
+         \DefineVerbatimEnvironment{Highlighting}{Verbatim}{breaklines,commandchars=\\\{\}}
+         % remove dots in toc
+         %\usepackage[titles]{tocloft}
+         %\renewcommand{\cftdot}{}
+         % remove table and figure caption numbers
+         \usepackage{caption}
+         \captionsetup{labelformat=empty}
+    include-before-body:
+      text: |
+        % also necessary to wrap long code lines
+        \RecustomVerbatimEnvironment{verbatim}{Verbatim}{
+          showspaces = false,
+          showtabs = false,
+          breaksymbolleft={},
+          breaklines
+          % Note: setting commandchars=\\\{\} here will cause an error 
+        }
+        % turn off page numbers
+        %\pagenumbering{gobble}
+        % roman section headers
+        \renewcommand{\thesection}{\Roman{section}}
+        \renewcommand{\thesubsection}{\arabic{subsection}}
+        \renewcommand{\thesubsubsection}{\thesubsection.\arabic{subsubsection}}
+        \renewcommand{\arraystretch}{1.5}
+---
+
+<!-- can be rendered with "quarto render toPDFviaQuarto.qmd --to pdf" -->
+
+{{< include README.md >}}
+
+\newpage
+
+## Appendix
```

Comment:
The Asterisk in some of the fields in the Appendix apparently stands for "mandatory", but it is not explained as far as I can see, within the Appendix.

Reply:
Good spotting. I added a sentence at the beginning of the appendix to explain that.